examples/code_llama/code_infilling_example.py

```diff
@@ -32,6 +32,7 @@ def main(
     enable_azure_content_safety: bool=False, # Enable safety check with Azure content safety api
     enable_sensitive_topics: bool=False, # Enable check for sensitive topics using AuditNLG APIs
     enable_salesforce_content_safety: bool=True, # Enable safety check with Salesforce safety flan t5
+    enable_llamaguard_content_safety: bool=False, # Enable safety check with Llama-Guard
```

Comment:
Same comment as the previous one. Do we need Llama Guard for code models?

Reply:
I just replied above, also to add that people can choose not to run the safety check at all in these scripts.